clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/api/KubernetesClientApiConverter.groovy

```diff
@@ -938,4 +939,26 @@ class KubernetesClientApiConverter {
 
     return daemonset
   }
+
+  static V1beta1StatefulSet toReplaceStatefulSet(
+                                                 V1beta1StatefulSet configuredStatefulSet,
+                                                 V1beta1StatefulSet deployedStatefulSet) {
+    //Based on 1.6 api model, spec: Forbidden: updates to statefulset spec for fields other than 'replicas', 'template', and 'updateStrategy' are forbidden."
```

Comment:
The current client supports 1.8

Reply:
Yes.  I can remove those notes for now.  I do need to test how kubernetes 1.8 model handle 1.6 k8s environment.